### PR TITLE
Freeze DAML-LF 1.dev into DAML-LF 1.5

### PR DESCRIPTION
**Pull request by hurryabit**
In other words, we release DAML-LF 1.5.

This is required for generic templates (#1387).

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Add a line to the [release notes](https://github.com/digital-asset/daml/blob/master/docs/source/support/release-notes.rst), if appropriate

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

---
This change is [Reviewable](https://reviewable.io/reviews/digital-asset/daml/1408)

### CODE REVIEW

File: daml-lf/spec/daml-lf-1.rst [hunk 1]
```diff
@@ -1822,6 +1851,17 @@ as described by the ledger model::
        ⇓ᵤ
      Err "Exercise actors do not match"  ‖ E₁ ; (st; keys)
 
+     'tpl' (x : T)
+         ↦ { 'choices' { …, 'choice' ChKind Ch (y : τ) (z) : σ  'by' eₚ ↦ eₐ, … }, … }  ∈  〚Ξ〛Mod
```
Comment (bitonic) [inline daml-lf/spec/daml-lf-1.rst]: what's the `(z)`?

Comment (hurryabit) [reply daml-lf/spec/daml-lf-1.rst]: It's for the contract id of the contract the choice is exercised on, called `self` in the surface language. It's the same in the rule above.

**Comment by remyhaemmerle-da** [comment]
@hurryabit Could you take a looks at my commit ?

File: daml-lf/spec/daml-lf-1.rst [hunk 2]
```diff
@@ -2061,6 +2068,14 @@ Decimal functions
 
   Returns the decimal string representation of the decimal.
 
+* ``FROM_TEXT_DECIMAL : 'Text' → 'Optional' 'DECIMAL'``
+
+  Given a string representation of a decimal returns the decimal
+  wrapped in ``Some``. If the input does not match the regexp 
+  ``[+-]?[0-9]+(\.[0-9]+)?`` or if the result of the conversion
+  cannot be mapped into a decimal without lost of precision, returns None.
```
Comment (hurryabit) [inline daml-lf/spec/daml-lf-1.rst]: @remyhaemmerle-da It should be "loss of precision".

**Pull request merged**